app/javascript/packs/javascripts/answers/destroy.js

```diff
@@ -0,0 +1,2 @@
+$("#modal-window").find(".modal-content").html("<%= j (render 'destroy') %>");
+jQuery("#modal-window").modal();
```

Comment:
Please add a newline to the end of the file :construction_worker_man:
```suggestion
jQuery("#modal-window").modal();

```